generate_and_apply_patch.sh

```diff
@@ -6,11 +6,59 @@ BOOT=${1:-/boot}
 IRFS_HOOK="/etc/initramfs-tools/hooks/acpi_override.sh"
 cd `mktemp -d`
 
+###
+# If the tested version of iasl is not currently installed...
+if [ "$(iasl -v | grep -c 20181031)" -eq 0 ]; then
```

Comment:
Something like `if [[ "$(iasl -v | grep version | sed 's/.*version \(.*\)$/\1/')" -ge 20181031 ]]` would only install `iasl` if it's to old.
I installed mine via the AUR and it's at version [20181213](https://www.archlinux.org/packages/community/x86_64/acpica/).

Reply:
Oh, interesting. I'll have to play around with that in my local terminal. I haven't got around to using `sed` in Bash yet. Thank you so much for code reviewing my PR!